x-pack/plugins/observability_solution/slo/public/pages/slo_details/slo_details.tsx

```diff
@@ -152,16 +152,16 @@ function getBreadcrumbs(
 ): ChromeBreadcrumb[] {
   return [
     {
-      href: basePath.prepend(paths.observability.slos),
-      text: i18n.translate('xpack.observability.breadcrumbs.slosLinkText', {
+      href: basePath.prepend(paths.slos),
+      text: i18n.translate('xpack.slo.breadcrumbs.slosLinkText', {
         defaultMessage: 'SLOs',
       }),
       deepLinkId: 'observability-overview:slos',
```

Comment:
@CoenWarmer `observability-overview` is the applicationId for the observability plugin. Since slo is now moved to its own plugin, do I need to keep observability-overview? Do you know if it is needed for application usage purposes? Let's catch up on `Application usage` and `Telemetry`, cause I was tracking down the changes you did on exploratory-view regarding telemetry [here](https://github.com/elastic/kibana/pull/153852/commits/55b8c29880ef7819509dc203ab33850d7aa9da25).

Reply:
Could it be that [this](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/exploratory_view/public/components/shared/exploratory_view/utils/telemetry.test.tsx#L51) is a leftover? Since exploratory-view is moved in a new plugin, I would expect it to be `exploratory-view` and not `observability-overview`, but maybe I miss something. Let's catch up on this.